### ISSUE TITLE
API not returning access token

### ISSUE THREAD

**Issue by Manu10744** [spotify/web-api#1493]
Issue found on March 04 2020.

#### Endpoint(s):
* `GET /authorize`
* `POST /api/token`

#### Steps to reproduce:
1. Send a GET Request to `/authorize` with the application `client_id` together with scopes and all other required params
2. Get authorize code and send a POST request to `/api/token` with required `grant_type` , `code` and `redirect_uri` params in the request body. Also, put `client_id` and `client_secret` of the application into the request body.

#### Expected behaviour:
API returns JSON data with an access token and other information.

#### Actual behaviour:
API returns `text/html` saying something went wrong, concrete error is not given.

I did it first with JavaScript and then with Postman:
![spotifyhelp](https://user-images.githubusercontent.com/38779302/75902120-d9e78200-5e3f-11ea-8871-8824760aeb17.jpg)

This is being returned:
![spotifyhelp2](https://user-images.githubusercontent.com/38779302/75902140-e075f980-5e3f-11ea-9e6a-46fa7ac9aae2.jpg)

As far as i can see i did everything according to the API documentation. I used my auth code inside the POST request in the `code` parameter, the redirect uri is the same like the one i used while getting the auth code in the first step, `grant_type` is "authorization_code" and so on. What is wrong here?

Thanks in advance.

**Comment by mikaelho** [comment]
For what it is worth, all the details you mention seem to match the flow that works for me today. Does your `redirect_uri` match in the two requests?

**Comment by Manu10744** [comment]
Yeah, the `redirect_uri` matched. But it seems like it works now, although i did not take any changes in my code.
